libs/application/graphql/src/lib/client.ts

```diff
@@ -8,6 +8,7 @@ import {
 import { onError } from '@apollo/client/link/error'
 import { RetryLink } from '@apollo/client/link/retry'
 import { setContext } from '@apollo/client/link/context'
+import { userManager } from '../../../../../apps/application-system/form/src/utils/userManager'
```

Comment:
I think this import needs to be done differently. Maybe the userManager needs to be in a lib